external/openglcts/modules/gl/gl3cCullDistanceTests.cpp

```diff
@@ -1138,7 +1141,7 @@ void CullDistance::FunctionalTest::buildPO(glw::GLuint clipdistances_array_size,
 
 	static const glw::GLchar* core_functionality = "#version 450\n";
 
-	static const glw::GLchar* extention_functionality = "#version 440\n"
+	static const glw::GLchar* extention_functionality = "#version 150\n"
```

Comment:
Since the KHR-GL33.cull_distance.* family of tests use tessellation shaders you'll need to add in some #extensions after dropping the version to 150. At the very least you'll need GL_ARB_tessellation_shader.

Reply:
Thank you, will take a look. I must not have re-run this with a GL4+ driver. I do have access to such hardware though, so should be able to work it out.